exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -658,11 +657,21 @@ private boolean tableModified(List<String> directories, Path metaFilePath,
     return false;
   }
 
+  /**
+   * Basic class for parquet metadata. Inheritors of this class are json serializable structures of
+   * different versions metadata cache files.
+   *
```

Comment:
Sorry, nit. This is Javadoc and is HTML formatted. Please insert <p> to separate paragraphs.